docs/operation/cluster_and_service_configuration.rst

```diff
@@ -195,9 +257,10 @@ then be applied with Salt.
 .. parsed-literal::
 
    root\@bootstrap $ kubectl exec --kubeconfig /etc/kubernetes/admin.conf \\
-                      -n kube-system -c salt-master salt-master-bootstrap -- \\
-                      salt-run state.sls \\
-                      metalk8s.addons.nginx-ingress.deployed \\
+                      $(kubectl --kubeconfig /etc/kubernetes/admin.conf \\
+                      get pod -n kube-system -l app=salt-master -o name) \\
+                      -n kube-system -c salt-master -- \\
+                      salt-run state.sls metalk8s.addons.nginx-ingress.deployed \\
```

Comment:
I think you want to apply both states in case control-plane is modified

```suggestion
                      salt-run state.sls metalk8s.addons.nginx-ingress.deployed,metalk8s.addons.nginx-ingress-control-plane.deployed \\
```